Custom Fees Acceptance Tests not using ObjectToStringSerializer

### Description

@Web3 acceptance tests fail: 
```
Caused by: org.hibernate.HibernateException: java.lang.IllegalArgumentException: The given string value: [{"amount": 10, "collector_account_id": 1051, "denominating_token_id": null, "all_collectors_are_exempt": false}] cannot be transformed to Json object```

### Steps to reproduce

1. Run @web3 acceptance tests.

### Additional context

_No response_

### Hedera network

other

### Version

v0.87.0

### Operating system

None

--- a/hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/store/accessor/CustomFeeDatabaseAccessor.java
+++ b/hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/store/accessor/CustomFeeDatabaseAccessor.java
@@ -20,12 +20,14 @@ import static com.hedera.mirror.web3.evm.utils.EvmTokenUtils.EMPTY_EVM_ADDRESS;
 import static com.hedera.mirror.web3.evm.utils.EvmTokenUtils.toAddress;
 import static java.util.Objects.requireNonNullElse;
 
+import com.hedera.mirror.common.converter.ObjectToStringSerializer;
 import com.hedera.mirror.common.domain.entity.EntityId;
 import com.hedera.mirror.web3.repository.CustomFeeRepository;
 import com.hedera.node.app.service.evm.store.contracts.precompile.codec.CustomFee;
 import com.hedera.node.app.service.evm.store.contracts.precompile.codec.FixedFee;
 import com.hedera.node.app.service.evm.store.contracts.precompile.codec.FractionalFee;
 import com.hedera.node.app.service.evm.store.contracts.precompile.codec.RoyaltyFee;
+import jakarta.annotation.PostConstruct;
 import jakarta.inject.Named;
 import java.util.ArrayList;
 import java.util.Collections;
@@ -38,6 +40,14 @@ import org.springframework.util.CollectionUtils;
 @Named
 @RequiredArgsConstructor
 public class CustomFeeDatabaseAccessor extends DatabaseAccessor<Object, List<CustomFee>> {
+
+    @PostConstruct
+    private void initConversion() {
+        // Ensure the ObjectToStringSerializer is instantiated.
+        // It is used for the conversion of Jsonb data into domain objects.
+        ObjectToStringSerializer.INSTANCE.toString();
+    }
+
     private final CustomFeeRepository customFeeRepository;
 
     private final EntityDatabaseAccessor entityDatabaseAccessor;

--- a/hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/Web3IntegrationTest.java
+++ b/hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/Web3IntegrationTest.java
@@ -16,7 +16,6 @@
 
 package com.hedera.mirror.web3;
 
-import com.hedera.mirror.common.converter.ObjectToStringSerializer;
 import com.hedera.mirror.common.domain.DomainBuilder;
 import com.hedera.mirror.web3.config.IntegrationTestConfiguration;
 import io.micrometer.core.instrument.MeterRegistry;
@@ -36,12 +35,6 @@ import org.springframework.test.context.jdbc.Sql;
 @SpringBootTest
 @Import(IntegrationTestConfiguration.class)
 public abstract class Web3IntegrationTest {
-
-    static {
-        // Ensure the ObjectToStringSerializer is instantiated for testing.
-        // It is used for the conversion of Jsonb data into domain objects.
-        ObjectToStringSerializer.INSTANCE.toString();
-    }
 
     protected final Logger log = LoggerFactory.getLogger(getClass());
 

